documentation/docs/14-configuration.md

```diff
@@ -5,6 +5,9 @@ title: Configuration
 Your project's configuration lives in a `svelte.config.js` file. All values are optional. The complete list of options, with defaults, is shown here:
 
 ```js
+// used by vite-plugin-svelte (https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/config.md)
+const vitePluginSvelteConfig = {};
```

Comment:
Having an object, and then spreading feels odd to me. Currently, top-level properties like `preprocess` and `compilerOptions` are used by `vite-plugin-svelte` too, so the separation may be a bit misleading. I think we should remove this line and the spreading expression below, and then...